package.json

```diff
@@ -9,7 +9,9 @@
   "author": "Bene Stangl",
   "license": "MIT",
   "dependencies": {
-    "request": "^2.83.0",
-    "underscore": "^1.8.3"
+    "graphql": "^15.6.1",
+    "graphql-request": "^3.6.1",
+    "request": "^2.88.2",
```

Comment:
also not used anymore i guess

Reply:
yep, that should have been removed on the last commit 🤷🏿